### PR TITLE
ci: validate commit message

### PR DESCRIPTION
**Pull request by csutorasr**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -103,6 +103,15 @@ jobs:
...
             - "~/bazel_repository_cache"
 
+  validate_commit_message:
+    <<: *job_defaults
+    steps:
+      - checkout:
+          <<: *post_checkout
+      - restore_cache:
+          key: *cache_key
+      - run: cd tools/validate-commit-message && $(npm bin)/jasmine
```
Comment (alexeagle) [inline .circleci/config.yml]: why don't you just run this as part of the lint job on line 60? we have already installed all npm dependencies there.

Comment (alexeagle) [inline .circleci/config.yml]: how does this ensure we only check the commit messages included in the PR, not all of the history?

Comment (gkalpak) [reply .circleci/config.yml]: I am not sure what the intention was, but currently this job only runs the unit tests for the `validate-commit-message` script. It doesn't validate any commit messages.
(Also, it doesn't depend on `build` afaict.)

Comment (csutorasr) [reply .circleci/config.yml]: Oh I see. Sorry.

**Pull request closed without merging**